harmonica/_io/geosoft_gxf_io.py

```diff
@@ -0,0 +1,352 @@
+# Function to read in GXF files, provided by USGS Generally
+# No guarantee this will read in your specific GXF files
+
+# Written by Thomas Martin, Unidata
+# Inspired by this gist: https://gist.github.com/jobar8/683483df605a906fb3da747b64627305
```

Comment:
The gist is released under a BSD License, so we should acknowledge the author and specify the license.

```suggestion
# Inspired by this gist: https://gist.github.com/jobar8/683483df605a906fb3da747b64627305
# created by Joseph Barraud and made available under the BSD License.
```